src/documents/tutorial-usingStringTemplates/UsingStringTemplates.md

```diff
@@ -0,0 +1,296 @@
+---
+title: Using String Templates
+layout: default
+---
+
+# Using String Templates #
+
+In the Infusion [Preference Framework](../PreferencesFramework.md), a "string template" is a mechanism for resolving variables within a string.
```

Comment:
I would say "resolving (interpolating)" for clarity - http://en.wikipedia.org/wiki/String_interpolation